api/src/pcapi/core/educational/api/institution.py

```diff
@@ -57,3 +60,36 @@ def update_educational_institution_data(
     for key, value in institution_data.items():
         setattr(educational_institution, key, value)
     return educational_institution
+
+
+def compare_adage_and_institution(ansco: str) -> None:
+    institutions = list(educational_repository.get_all_educational_institutions()[0])
+    page = 1
+    adage_institutions_total = []
+    adage_institutions = None
+    while not adage_institutions:
+        adage_institutions = list(adage_client.get_adage_educational_institutions(ansco=ansco, page=page))
+        adage_institutions_total.extend(adage_institutions)
+        page += 1
+    synchronise_adage_and_institution(institutions=institutions, adage_institutions=adage_institutions_total)
+
+
+def synchronise_adage_and_institution(institutions: list, adage_institutions: list) -> None:
+    institutions_return_by_adage = {adage_institution.uai for adage_institution in adage_institutions}
+
+    for institution in institutions:
+        if institution.institutionId in institutions_return_by_adage:
+            institution.isActive = True
+            adage_institution = adage_institutions[int(institution.institutionId) - 1]
+            institution.name = adage_institution.libelle
+            institution.city = adage_institution.communeLibelle
+            institution.postalCode = adage_institution.codePostal
+            institution.email = adage_institution.courriel
+            institution.phoneNumber = adage_institution.telephone
+            institution.institutionType = INSTITUTION_TYPES.get(adage_institution.sigle)
```

Comment:
un parametre par defaut serait plus sage ici.
```suggestion
            institution.institutionType = INSTITUTION_TYPES.get(adage_institution.sigle,  institution.institutionType)
```